### DISCUSSION TITLE
How to enable pen pressure in relative mode (mouse mode)?

### DISCUSSION THREAD

**Discussion by Belamat** [OpenTabletDriver/OpenTabletDriver#2456] [General]
Im on Linux Fedora, with an xp pen fun xs, when i enable relative mode the pressure sensitivity dissapears. Only available in artist mode, but then i lose relative mode that im using. Any clue? thanks!

**Comment by jamesbt365** [comment]
That's how it works yes. Artist mode is the only mode on Linux that exposes pressure. And there's no relative mode of that because of some weird things

**Reply by Belamat** [reply]
Thanks James, could you elaborate more on that weird things you mentioned?

**Reply by gonX** [reply]
Among the weird things is the relative mode emulation necessary to make a tablet device behave relatively, described in my response on InfinityGhost's comment.

**Comment by InfinityGhost** [comment]
> Im on Linux Fedora, with an xp pen fun xs, when i enable relative mode the pressure sensitivity dissapears. Only available in artist mode, but then i lose relative mode that im using. Any clue? thanks!

This likely could be implemented but depends on how libinput and/or your window manager handles relative pointers with pressure sensitivity.

**Reply by Belamat** [reply]
Ho it would be great to implement that feature! Could you tell me where to look for trying to implement that? 
Thanks!

**Reply by gonX** [reply]
You would have to duplicate (or extend) the current artist mode functionality to allow relative reporting. OTD would have to emulate the relative portion itself, rather than exposing it as a mouse as this is how relative mode works now. Mouse devices cannot report pressure sensitivity under Linux, and tablet devices must report absolute coordinates.

**Reply by gonX** [reply]
With that bring said I don't think this is a good first PR

**Reply by Belamat** [reply]
ok thanks, it doesn't sound that simple, do you see this being implemented in the future?
